client/src/amplify-ui/sidebar-layout/sidebar-close-button/sidebar-close-button.style.ts

```diff
@@ -2,11 +2,15 @@ import {css} from "emotion";
 import {MQTablet} from "../../styles/media";
 
 export const sidebarCloseButtonStyle = css`
-  margin-right: 0.3em;
-  margin-top: 0.3em;
-  position: sticky;
+  margin: 0.75rem 0.75rem 0 0;
+
+  user-select: none;
```

Comment:
Why would we want users not be able to select ?

Reply:
My intuition says that users shouldn't ever be able to select UI elements that aren't text.  It's selectable just because it's an image, which to me breaks the illusion of how the interface should essentially fade away to showcase the content.  Anyway, it's a good callout.  Not sure if it's worth the lines of CSS and potential inconsistency on the site

Reply:
How will this impact accessibility? Will the aria-labels still be read?

Reply:
Good callout @renebrandel, @tjleing, can we look into this?

Reply:
Should be resolved with the next commit